policies/travel-policy.md

```diff
@@ -82,7 +83,7 @@ in the ***Travel Approver Table***.
   | Travel Approver Table | Total Value of Travel |
   | --- | --- |
   | Manager and OMC member | US\$15,000 |
-  | Travel over US\$15,000 can only get approval direct from the OMC |  
+  | Travel over US\$15,000 can only get approval direct from the OMC |
```

Comment:
Matt: This means there are 7 different travel approvers. That seems excessive for an organisation of our size.
Koca: I understand that it should be approved by a manager and at least one OMC member.

Reply:
That's not the way I read the policy. Above it says "authorized either by the OMC or a designated Travel Approver"....note that it is "a designated Travel Approver" which implies just one. I interpret the table as meaning Manager and OMC members can all approve travel up to the value of $15,000. I don't see any need for OMC members individually being able to approve travel. The only issue is when the Manager themselves needs approval for travel - but in that case it can be approved collectively by the OMC.

Reply:
OK, will propose an update in that sense.